src/components/NavigationTabs/index.tsx

```diff
@@ -82,7 +84,7 @@ export function AddRemoveTabs({
 
   return (
     <Tabs>
-      <RowBetween style={{ padding: '1rem 1rem 0 1rem' }}>
+      <RowBetween style={{ padding: '1rem 1rem 0 1rem' }} align="center">
```

Comment:
it may look misaligned bc Settings were removed -- should look proper once settings are back?